src/WebJobs.Extensions.ApiHub/ApiHubFileTriggerAttribute.cs

```diff
@@ -13,11 +14,13 @@ public sealed class ApiHubFileTriggerAttribute : ApiHubFileAttribute
         /// </summary>
         /// <param name="key">App settings key name that have the connections string</param>
         /// <param name="path">Relative path to the file <example>/folder/subfolder/file.txt</example></param>
+        /// <param name="fileWatcherType">Type of the file watcher.</param>
         /// <param name="pollIntervalInSeconds">The poll interval in seconds.</param>
-        public ApiHubFileTriggerAttribute(string key, string path, int pollIntervalInSeconds = 0)
+        public ApiHubFileTriggerAttribute(string key, string path, FileWatcherType fileWatcherType = FileWatcherType.Created, int pollIntervalInSeconds = 0)
```

Comment:
Where is FileWatcherType defined? ApiHub defined its own enum? In FileTriggerAttribute (source [here](https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions/Extensions/Files/FileTriggerAttribute.cs#L37)) I used the core WatcherChangeTypes flags enum.